examples/paycrest/.env.example

```
@@ -0,0 +1,23 @@
+# AVNU's paymaster — the wallet pays no STRK gas. Unset for the SDK's
+# default `user_pays` mode.
+AVNU_PAYMASTER_API_KEY=
+
+# Optional gas token within sponsored mode: USDT | USDC | STRK | ETH.
+# Leave empty for fully sponsored (paymaster covers all gas). Requires
+# AVNU_PAYMASTER_API_KEY to be set — otherwise the example errors.
+PAYCREST_GAS_TOKEN=
+
+# Recipient bank account (NGN example via GTB)
+RECIPIENT_INSTITUTION=KUDANGPC
+RECIPIENT_ACCOUNT_IDENTIFIER=2002948489
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RECIPIENT_ACCOUNT_IDENTIFIER key should go before the RECIPIENT_INSTITUTION key

(UnorderedKey)